### DISCUSSION TITLE
Open-webui with vLLM

### DISCUSSION THREAD

**Discussion by wolfgangsmdt** [open-webui/open-webui#6956] [General]
Is there any idea on how to run vLLM inference engine with Open-webui chat interface instead of ollama?

**Comment by jay2jp** [comment]
I'm pretty sure you can just use the openai compatible server that vllm offers. [vllmDocs](https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html) From there you can just connect it like its an openai server under connections in the admin panel .

**Comment by DJJones66** [comment]
When I played with it this is what I found and used:
https://github.com/open-webui/open-webui/discussions/508

**Comment by wolfgangsmdt** [comment]
@jay2jp @DJJones66 

Thank you for the reply.
I tried to run the openai api vllm with open-webui but I have not found my model neither I can send prompts.
[You can find here what I did](https://github.com/vllm-project/vllm/issues/10322#issuecomment-2476299104) 

I know I am so close to the solution but I need little help to figure out why it did not worked for me.

**Comment by wolfgangsmdt** [comment]
thank you very much @jay2jp  and @DJJones66 

It worked as you suggested with openai api with adding `--chat-template` 
here is the full thread : https://github.com/vllm-project/vllm/issues/10322
